test/unit/spec/v2/view-builder/views/IdentifierView_spec.js

```diff
@@ -90,4 +90,19 @@ describe('v2/view-builder/views/IdentifierView', function() {
     expect(testContext.view.$el.find('.o-form-fieldset-container .sign-in-with-idp').length).toEqual(0);
     expect(testContext.view.$el.find('.o-form-button-bar .sign-in-with-idp').length).toEqual(0);
   });
+
+  it('view renders IDP buttons correctly with tooltip', function() {
```

Comment:
so button tooltip will always exists now ? I think make it available only when there is a long name..

Reply:
@pradeepdewda-okta, yes for now that's the behavior. If we want to dynamically add a tooltip, then we'd need to come up with a character limit that would be sensible for all languages past which we would generate that. I didn't go down that route because this was the safer approach (I didn't know about the screen reading issue). 

Any ideas on how best to come up with this threshold?

Reply:
how about we do this based on overlapping text? If text overlaps the button width then add a tooltip ?

Reply:
@pradeepdewda-okta @edburyenegren-okta I just pushed some updates to only add the tooltip when the text is longer than the allowed width. Please have a look.